src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs

```diff
@@ -146,7 +146,14 @@ public async Task GracefulTurnsAbortiveIfRequestsDoNotFinish()
 
                 await closingMessageTask;
                 cts.Cancel();
-                await stopServerTask;
+                try
+                {
+                    await stopServerTask;
+                }
+                catch (OperationCanceledException)
```

Comment:
@halter73 @Tratcher Do we want to add a comment about why we're doing this for future reference?
(Also for the MaxRequestBufferSizeTests one)